.doc_gen/cross-content/phrases-code-examples.ent

```diff
@@ -25,6 +25,13 @@
 <!ENTITY guide-rust-dev '&Rustlong; developer guide'>
 <!ENTITY guide-rust-api '&Rustlong; API reference'>
 
+<!-- SDK for Bash (a virtual SDK that uses the CLI) -->
+
+<!ENTITY Bashlong '&CLIlong; with Bash script'>
+<!ENTITY Bash '&CLI; with Bash script'>
+<!ENTITY guide-rust-dev '&guide-cli-ug;'>
```

Comment:
Is it supposed to say Rust here? I'm not sure what this is affecting, but might be a copy/paste error.

Reply:
Fixed (removed because these aren't needed anyway).